FreeBSD tests timed out right after the LibGit2 tests.

Travis 64bit Linux tests failed trying to install dependencies:

```
⋮
W: Failed to fetch http://ppa.launchpad.net/git-core/ppa/ubuntu/dists/trusty/main/i18n/Translation-en  Unable to connect to ppa.launchpad.net:http:
W: Failed to fetch http://ppa.launchpad.net/couchdb/stable/ubuntu/dists/trusty/main/binary-amd64/Packages  Unable to connect to ppa.launchpad.net:http:
W: Failed to fetch http://ppa.launchpad.net/couchdb/stable/ubuntu/dists/trusty/main/binary-i386/Packages  Unable to connect to ppa.launchpad.net:http:
W: Some index files failed to download. They have been ignored, or old ones used instead.

The command "sudo -E apt-get -yq --no-install-suggests --no-install-recommends --force-yes install ccache libssl1.0.0 bar time g++-5 gfortran-5" failed and exited with 100 during .
```

CircleCI x86_64 failed with:

```
Killed
*** This error is usually fixed by running `make clean`. If the error persists, try `make cleanall`. ***
Makefile:202: recipe for target '/home/circleci/project/usr/lib/julia/sys-debug-o.a' failed
make[1]: *** [/home/circleci/project/usr/lib/julia/sys-debug-o.a] Error 1
Makefile:81: recipe for target 'julia-sysimg-debug' failed
make: *** [julia-sysimg-debug] Error 2
make: *** Waiting for unfinished jobs....
Killed
*** This error is usually fixed by running `make clean`. If the error persists, try `make cleanall`. ***
Makefile:201: recipe for target '/home/circleci/project/usr/lib/julia/sys-o.a' failed
make[1]: *** [/home/circleci/project/usr/lib/julia/sys-o.a] Error 1
Makefile:78: recipe for target 'julia-sysimg-release' failed
make: *** [julia-sysimg-release] Error 2
Exited with code 2
```

All the other builds passed. I don't think these failures are relevant, but I'll request review before merging.

--- a/base/meta.jl
+++ b/base/meta.jl
@@ -188,7 +188,7 @@ emitted from a `@generated` function.
 
 The performed transformations are:
 
-- replace slot numbers in the range `1:slot_offset` with the corresponding items in `slot_replacements`
+- replace slot numbers in the range `1:length(slot_replacements)` with the corresponding items in `slot_replacements`
 - increment other slot numbers by `slot_offset`
 - substitute static parameter placeholders (e.g. `Expr(:static_parameter, 1)`) with the corresponding
 values in `static_param_values`

